<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InjectionWindow</class>
 <widget class="QMainWindow" name="InjectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonInject">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>430</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Inject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSelectedProcess">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;none&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>370</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Target Process:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetProcesses">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="itemAlignment">
     <set>Qt::AlignLeading</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProcessSearch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Search Processes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDLLPath">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>181</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DLL Path (x64)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonBrowse">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>61</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxInjectionModes">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>390</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>242</x>
      <y>370</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Method:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>249</x>
      <y>130</y>
      <width>103</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
